src/test/java/seedu/address/nusmods/DataFetcherManagerTest.java

```diff
@@ -125,14 +125,17 @@ public void fetchAndSaveModules_successfullySaveModuleInfoToFile()
 
         manager.fetchAndSaveModules();
         Map<String, ModuleInfo> actualMap = readModuleInfoMapFromFile(TEST_SAVE_FILE_PATH);
-        Optional<ModuleInfo> expectedModule = JsonUtil.readJsonFile(Paths.get(CS1010X_MODULE_INFO), ModuleInfo.class);
+
+        Optional<ModuleInfo> expectedCs1010x = JsonUtil.readJsonFile(Paths.get(CS1010X_MODULE_INFO), ModuleInfo.class);
+        Optional<ModuleInfo> expectedIs1103 = JsonUtil.readJsonFile(Paths.get(IS1103_MODULE_INFO), ModuleInfo.class);
 
         // clean up test data file after we're done
         Files.deleteIfExists(Paths.get(TEST_SAVE_FILE_PATH));
 
-        if (expectedModule.isPresent()) {
+        if (expectedCs1010x.isPresent() && expectedIs1103.isPresent()) {
```

Comment:
Updated this and added IS1103 cos its now part of the list of new required modules